scripts/master_init.sh

```diff
@@ -0,0 +1,220 @@
+#!/usr/bin/env bash
```

Comment:
I think this also belongs to `deploy/scripts` and should be also unified with what's in there now. There's suddenly a lot of shell scripts..